4-DeviceCodeFlow/Readme.md

```diff
@@ -0,0 +1,236 @@
+---
+services: active-directory
+platforms: dotnet
+author: v-shln
+level: 100
+client: .NET Desktop (Console)
+service: Microsoft Graph
+endpoint: Microsoft identity platform
+page_type: sample
+languages:
+  - csharp  
+products:
+  - azure
+  - azure-active-directory  
+  - dotnet
+  - office-ms-graph
+description: "This sample demonstrates a .NET Desktop (Console) application authenticating with device code flow"
+---
+# Sign-in a user with the Microsoft identity platform using the device code flow and call Microsoft Graph.
+
+![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/<BuildNumber>/badge)
+
+## About this sample
+
+### Overview
+
+This sample demonstrates a .NET Desktop (Console) application calling The Microsoft Graph.
+
+1. The .NET Desktop (Console) application uses the Microsoft Authentication Library (MSAL) to obtain a JWT access token from Azure Active Directory (Azure AD), using device code flow:
+2. The access token is used as a bearer token to authenticate the user when calling the Microsoft Graph.
+
+> Looking for previous versions of this code sample? Check out the tags on the [releases](../../releases) GitHub page.
+
+![Overview](./ReadmeFiles/topology.png)
```

Comment:
change text "device-profile" to "device code flow" in the image

Reply:
Updated the topology.
Thanks.